From 77bdcb9b291e6842de79b1d421aacff74283ba02 Mon Sep 17 00:00:00 2001
From: Peter Xu <peterx@redhat.com>
Date: Thu, 10 Mar 2016 14:37:33 +0800
Subject: [PATCH 059/134] trace-cmd: Install trace-cmd.bash when installing

Signed-off-by: Peter Xu <peterx@redhat.com>
Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 Makefile | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 73a98d144078..cfa4ffc8f673 100644
--- a/Makefile
+++ b/Makefile
@@ -68,6 +68,8 @@ endif
 HELP_DIR = -DHELP_DIR=$(html_install)
 HELP_DIR_SQ = '$(subst ','\'',$(HELP_DIR))'
 
+BASH_COMPLETE_DIR ?= /etc/bash_completion.d
+
 # copy a bit from Linux kbuild
 
 ifeq ("$(origin V)", "command line")
@@ -562,7 +564,10 @@ $(PYTHON_PY_PLUGINS): %.install : %.py force
 
 install_python: $(PYTHON_SO_INSTALL) $(PYTHON_PY_PROGS) $(PYTHON_PY_LIBS) $(PYTHON_PY_PLUGINS)
 
-install_cmd: all_cmd install_plugins install_python
+install_bash_completion: force
+	$(Q)$(call do_install_data,trace-cmd.bash,$(BASH_COMPLETE_DIR))
+
+install_cmd: all_cmd install_plugins install_python install_bash_completion
 	$(Q)$(call do_install,trace-cmd,$(bindir_SQ))
 
 install: install_cmd
-- 
2.5.5

